app/scss/base/base.scss

```diff
@@ -0,0 +1,31 @@
+body {
+  font-family: "Roboto", Sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  color: $dark-grey;
+  -webkit-font-smoothing: antialiased;
+  background-color: $background;
+
+  &.ReactModal__Body--open {
```

Comment:
Rename or remove this? 

Same goes for the React mentions in and around: `app/scss/components/confirmation_dialog.scss:67`

Reply:
I've removed all react-specific rules.